doc/users/whats_new/path_simplification_updates.rst

```diff
@@ -0,0 +1,8 @@
+Path simplification updates
+---------------------------
+
+Lines will now be simplified according to the ``path.simplify`` and
+``path.simplify_threshold`` RC parameters, as long as other conditions
+such as not using dashed lines are met. Simplification will only happen
+when drawing the line segment portion of a path, if you are also drawing
+markers you should consider using the ``markevery`` option to ``plot``.
```

Comment:
Can you add a remark here about reasonable values and why you would want to use this?

As near as I can tell this feature is un-documented except for a comment in the change log.